pkgs/tools/networking/zap/default.nix

```diff
@@ -1,30 +1,25 @@
-{ lib, stdenv, fetchFromGitHub, jdk8, ant, runtimeShell }:
+{ lib, stdenv, jdk8, bash, runtimeShell }:
 
-let jdk = jdk8; in
-stdenv.mkDerivation rec {
+let jdk = jdk8;
+in stdenv.mkDerivation rec {
   pname = "zap";
-  version = "2.7.0";
-  src = fetchFromGitHub {
-    owner = "zaproxy";
-    repo = "zaproxy";
-    rev =version;
-    sha256 = "1bz4pgq66v6kxmgj99llacm1d85vj8z78jlgc2z9hv0ha5i57y32";
+  version = "2.10.0";
+  src = fetchTarball {
+    url = "https://github.com/zaproxy/zaproxy/releases/download/v${version}/ZAP_${version}_Linux.tar.gz";
+    sha256 = "1877kas504vfx1ryn29brlf06lmwn47pmzaj87xxq5ibc376qll1";
   };
 
-  buildInputs = [ jdk ant ];
-
-  buildPhase = ''
-    cd build
-    echo -n "${version}" > version.txt
-    ant -f build.xml setup init  compile dist copy-source-to-build package-linux
-  '';
+  buildInputs = [ jdk bash ];
 
   installPhase = ''
-    mkdir -p "$out/share"
-    tar xvf  "ZAP_${version}_Linux.tar.gz" -C "$out/share/"
     mkdir -p "$out/bin"
+    mkdir -p "$out/share"
+
+    cp -pR . "$out/share/${pname}/"
+
     echo "#!${runtimeShell}" > "$out/bin/zap"
-    echo \"$out/share/ZAP_${version}/zap.sh\" >> "$out/bin/zap"
+    echo "cd $out/share/${pname}" >> "$out/bin/zap"
+    echo  "./zap.sh" >> "$out/bin/zap"
```

Comment:
```suggestion
    echo -e "cd $out/share/${pname}\n./zap.sh" >> "$out/bin/zap"
```

Reply:
Had to change it a bit (use  makeWrapper, so change is not applicable anymore) ;)